src/screens/community/screen/communityScreen.js

```diff
@@ -119,6 +132,7 @@ const CommunityScreen = ({ route }) => {
               selectedOptionIndex={_getSelectedIndex()}
               tag={tag}
               pageType="community"
+              handleOnScroll={isExpanded ? _handleOnScroll : null}
```

Comment:
if simple enough, can we use onScrollBeingDrag to hide header as soon scroll begin instead of hiding at end of scroll, not sure why but handleOnScroll hiding header after I raise my finger and stop scrolling. will be great to have reveal animation on scroll back to top, but can be a separate PR etc.